narwhal/network/src/metrics.rs

```diff
@@ -9,12 +9,42 @@ use prometheus::{
 use std::sync::Arc;
 use tracing::warn;
 
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub struct NetworkConnectionMetrics {
     /// The connection status of known peers. 0 if not connected, 1 if connected.
     pub network_peer_connected: IntGaugeVec,
     /// The number of connected peers
     pub network_peers: IntGauge,
+
+    /// PathStats
+    /// The rtt for a peer connection.
+    pub network_peer_rtt: HistogramVec,
```

Comment:
Using an IntGaugeVec seems better. Since this metric is labeled with peer ID, I don't expect taking unrelated samples of rtt.